tests/backend/topology/conftest.py

```diff
@@ -27,8 +27,15 @@ def swarm():
     return Swarm(**attrs_at_t)
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
 def k():
     """Default neighbor number"""
     _k = 1
     return _k
+
+
+@pytest.fixture(scope="module", params=[True, False])
```

Comment:
Hmmm, I wonder why we need a fixture for this? See my next comment and check if it works :smile: